yarn.lock

```diff
@@ -0,0 +1,3159 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Would you mind removing this file from the commit?

Reply:
Addressed in https://github.com/ConsenSys/solc-typed-ast/pull/98/commits/19f312848b466885b4cdcb3fff2eb5dd7ec6c813